packages/localization/src/languages/en.json

```diff
@@ -381,11 +381,16 @@
                     "canEditUserRoles": "Give and edit user roles",
                     "canPlaceOrder": "Place orders on the exchange",
                     "canBuyIRec": "Directly buy I-RECs",
+                    "canBuyCertificates": "Directly buy certificates",
                     "canCreateAndBuyIRecBundles": "Create and buy I-REC bundles",
+                    "canCreateAndBuyCertificateBundles": "Create and buy certificate bundles",
```

Comment:
Instead of duplicating these localization strings, please create modular strings similar to https://github.com/energywebfoundation/origin/blob/028236a0159edf4b8c850fdbf52bd740b7480fa0/packages/localization/src/languages/en.json#L86

So instead of 
```
"canCreateAndBuyIRecBundles": "Create and buy I-REC bundles",
"canCreateAndBuyCertificateBundles": "Create and buy certificate bundles",
```
do

```
"canCreateAndBuyCertificateBundles": "Create and buy {certificateType} bundles",
```

And then in the code:
```
t('user.feedback.roleChanged.canCreateAndBuyCertificateBundles', {
    certificateType: features.includes(OriginFeature.IRec) ? 'I-REC' : 'certificate'
})
```